Changes.md

```diff
@@ -1,6 +1,16 @@
 0.61.x.x (relative to 0.61.13.1)
 =========
 
+Improvements
+------------
+
+- PathListingWidget :
+  - Added an option to `selectionMode` allowing single and multiple cell selections.
```

Comment:
This belongs in the API section. And maybe the interaction descriptions belong in the LightEditor section, since that's the only place users will see them right now?